team.html

```diff
@@ -97,6 +97,28 @@ <h3>Dr. Sumam David</h3>
                 </div>
             </div>
         </div>
+        <div class="row">
```

Comment:
So just adding it like this messes up the alignment - Copy the entire Faculty Members section from [https://github.com/IEEE-NITK/ieee-nitk.github.io/blob/master/team.html](https://github.com/IEEE-NITK/ieee-nitk.github.io/blob/master/team.html)